partiql/Cargo.toml

```diff
@@ -20,6 +20,25 @@ edition.workspace = true
 [lib]
 bench = false
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
 [dependencies]
+partiql-parser = { path = "../partiql-parser" }
+partiql-ast = { path = "../partiql-ast" }
+partiql-value = { path = "../partiql-value" }
+partiql-logical = { path = "../partiql-logical" }
+partiql-logical-planner = { path = "../partiql-logical-planner" }
+partiql-eval = { path = "../partiql-eval" }
```

Comment:
May need to add version numbers. For the 0.2.0 (https://github.com/partiql/partiql-lang-rust/pull/272/files), I wasn't able to publish without adding the version numbers as part of the dependencies.